stacksCollectible/index.ts

```diff
@@ -90,16 +86,7 @@ async function fetchNftData(
     };
 
     if (contractInfo[1] === 'bns') {
-      // no further data required for BNS, arrange into collection
-      // currently stacks only supports 1 bns name per address
-      const bnsCollection: StacksCollectionData = {
-        collection_id: contractId,
-        collection_name: 'BNS Names',
-        total_nft: 1,
-        thumbnail_nfts: [nftData],
-        all_nfts: [nftData],
-      };
-      collectionRecord.bns = bnsCollection;
+      continue;
```

Comment:
Are we getting rid of BNS names?

If so, it would also be nice to just have `if (contractInfo[1] !== 'bns')` and get rid of the continue and the else

Reply:
Ok, I see you're using it in the organise method. I think this if else shouldn't actually exist. It should always do what is currently in the else.